checks/check_sample

```diff
@@ -36,38 +36,68 @@
 # CHECK_DOC_checkN=""
 # CHECK_CAF_EPIC_checkN=""
 
+# General comments
+# ----------------
+# Do not add double quotes (") arround variable ${PROFILE_OPT}
```

Comment:
```suggestion
# Do not add double quotes (") arround variable ${PROFILE_OPT} because this variable holds "--profile <profile-name>" and we need to read it as it is.
```